Makefile

```diff
@@ -13,35 +13,42 @@
 # limitations under the License.
 
 REGISTRY_NAME = quay.io/k8scsi
-IMAGE_VERSION = v0.2.0
+IMAGE_VERSION = canary
 
-.PHONY: all flexadapter nfs hostpath iscsi cinder clean
+.PHONY: all flexadapter nfs hostpath iscsi cinder clean hostpath-container
 
 all: flexadapter nfs hostpath iscsi cinder
 
 test:
 	go test github.com/kubernetes-csi/drivers/pkg/... -cover
 	go vet github.com/kubernetes-csi/drivers/pkg/...
 flexadapter:
-	if [ ! -d ./vendor ]; then dep ensure; fi
+	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o _output/flexadapter ./app/flexadapter
 nfs:
-	if [ ! -d ./vendor ]; then dep ensure; fi
+	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o _output/nfsplugin ./app/nfsplugin
-livenessprobe:
-	if [ ! -d ./vendor ]; then dep ensure; fi
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o _output/livenessprobe ./app/livenessprobe/cmd
 hostpath:
-	if [ ! -d ./vendor ]; then dep ensure; fi
+	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o _output/hostpathplugin ./app/hostpathplugin
-hostpath-container: hostpath livenessprobe
-	docker build -t $(REGISTRY_NAME)/hostpathplugin:$(IMAGE_VERSION) -f ./pkg/hostpath/extras/docker/Dockerfile .
+hostpath-container: hostpath _output/livenessprobe
```

Comment:
It needs to be reverted as liveness probe now sits in a separate repo and it will not be a part of plugin image.